lms/static/js/demographics_collection/DemographicsCollectionModal.jsx

```diff
@@ -399,7 +402,7 @@ class DemographicsCollectionModal extends React.Component {
                       {gettext("What is the highest level of education that any of your parents or guardians have achieved?")}
                     </label>
                     <select
-                      className="form-control pb-3"
```

Comment:
Removing these as they are already on the surrounding `div`

Reply:
Good catch! I completely missed those when I was cleaning up that last PR.